### PR TITLE
#166005764 Fix invalid url

### PR DESCRIPTION
**Pull request by joaquinto**
#### What does this PR do?
Handle invalid url
#### Description of Task to be completed?
Have the server handle invalid url properly
#### How should this be manually tested?
After cloning this repository, cd into it and run npm test in the terminal or command prompt
#### What are the relevant pivotal tracker stories?
#166005764

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -27,15 +27,12 @@ app.use('/api/v1/', repayments);
 
 app.use('/docs-api', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
 
+app.use((req, res) => {
```
Comment (frostyblok) [inline server/index.js]: I would like to know how you intend to catch routing errors with this. Also, I would love to know the reasoning behind this implementation.

**Review by frostyblok — COMMENTED** [review]
Good job attending to the feedback @joaquinto. Please attend to the comment below

File: server/index.js (hunk 1 above)
Comment (frostyblok) [reply server/index.js]: I am still awaiting your response

**Comment by joaquinto** [comment]
Oh! I'm very sorry. I wanted to effect the changes with that off my UI but
I will do that now.

**Review by frostyblok — APPROVED** [review]

**Pull request merged**